src/utils/slangHelper.ts

```diff
@@ -123,8 +123,17 @@ export const externalBuiltIns = {
  * provides the original function with the required
  * externalBuiltIns, such as display and prompt.
  */
-export function createContext<T>(chapter: number, externals: string[], externalContext: T) {
-  return createSlangContext<T>(chapter, externals, externalContext, externalBuiltIns);
+export function createContext<T>(
+  chapter: number,
+  variant: Variant | undefined,
```

Comment:
The `variant` parameter can be undefined, since the variant property is currently optional on the Library type